config/vendor-api-v1.yml

```diff
@@ -257,6 +257,50 @@ paths:
           "$ref": "#/components/responses/NotFound"
         '422':
           "$ref": "#/components/responses/UnprocessableEntity"
+  "/reference-data/gcse-subjects":
+    get:
+      tags:
+      - Reference data
+      responses:
+        '200':
+          description: An array of possible GCSE subjects. Candidates are offered these in an autocomplete when filling in the form, but are also able to provide free-text responses.
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ListResponse"
+  "/reference-data/gcse-grades":
+    get:
+      tags:
+      - Reference data
+      responses:
+        '200':
+          description: An array of possible GCSE grades including double-award grades.
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ListResponse"
+  "/reference-data/a-and-as-level-subjects":
+    get:
+      tags:
+      - Reference data
+      responses:
+        '200':
+          description: An array of possible A and AS level subjects. Candidates are offered these in an autocomplete when filling in the form, but are also able to provide free-text responses.
```

Comment:
By grouping these together, we don't mention that a candidate can't have an A* grade at AS level. Should this be noted somewhere?

Reply:
Good spot — guilty as charged. I saw the two sets of possible grades were slightly different but decided that it was probably simpler for everyone if we had `as-and-a-level-grades` to go with `as-and-a-level-subjects`, rather than separate `as-level-grades` and `a-level-grades`. If anyone complains I'll split them!